test-case/check-pause-release-suspend-resume.sh

```diff
@@ -146,7 +143,7 @@ expect {
         }
 
         #enter suspend-resume cycle once per pause instance
-        set retval [catch { exec bash check-suspend-resume.sh -l 1 } msg]
+        set retval [catch { exec bash $(dirname "${BASH_SOURCE[0]}")/check-suspend-resume.sh -l 1 } msg]
```

Comment:
Please define a MYDIR or TOPDIR or TESTDIR variable at the top and do the `dirname` dance only once at the top. See examples in other scripts.